fluentd-modsecurity/.gitlab-ci.yml

```diff
@@ -0,0 +1,34 @@
+Lint fluentd-modsecurity:
+  stage: "Quality"
+  image: hadolint/hadolint:v1.17.5-debian
+  script:
+    - hadolint ./fluentd-modsecurity/Dockerfile
+
+Build fluentd-modsecurity:
+  stage: "Build"
+  needs:
+    - Lint fluentd-modsecurity
+  extends: .base_register_to_gitlab_stage
+  variables:
+    CONTEXT: fluentd-modsecurity
+    IMAGE_NAME: ${CI_REGISTRY_IMAGE}/fluentd-modsecurity
+
+Test fluentd-modsecurity:
+  stage: "Test"
+  needs:
+    - Lint helm
+    - Build helm
+  image: "${CI_REGISTRY_IMAGE}/fluentd-modsecurity:${CI_COMMIT_SHA}"
+  environment: feature-dev
```

Comment:
Does the `Test fluentd-modsecurity` require a connection to the cluster ?

Reply:
Non on peut le supprimer